app/uk/gov/hmrc/economiccrimelevyaccount/models/FinancialData.scala

```diff
@@ -230,7 +190,6 @@ object FinancialDataDocumentType {
           case "TRM Amend Charge"  => JsSuccess(AmendedCharge)
```

Comment:
Literals are being used twice, which indicates they should be extracted out into private vals